index.js

```diff
@@ -407,12 +407,6 @@ let rcl = {
   },
 };
 
-process.on('SIGINT', () => {
-  debug('Catch ctrl+c event and will cleanup and terminate.');
-  rcl.shutdown();
-  process.exit(0);
```

Comment:
The initial purpose of installing a handler and handling SIGINT here is that I once found the program cannot be terminated by `ctrl+c` sometimes and this gives a chance to quit the process. I'm not sure if we may still meet the issue.

Reply:
Finally, I remember the reason why we need to shut down here (considering the code was written several years ago...).

At that time, there is no kind of Context concept, so `rcl.shutdown()` will stop the sub-thread and the main thread could quit as expected. But now user may create more than one Context, which means more sub-threads are running simultaneously, so simply executing `rcl.shutdown()` is not enough and we must ensure all contexts are cleaned up properly when terminating.
https://github.com/RobotWebTools/rclnodejs/blob/d41f7e91510cd55ff672dad8f7f0be095ed8c00c/index.js#L305